tgtg_scanner/models/config.py

```diff
@@ -244,6 +244,7 @@ class SMTPConfig(NotifierConfig):
     use_ssl: bool = False
     sender: Union[str, None] = None
     recipients: list[str] = field(default_factory=list)
```

Comment:
Maybe allow no recipients if recipients per item is set?

Reply:
I’d stick to `Recipients` being mandatory to have a fallback just in case. Checking the favorited items against the `RecipientsPerBasket` Json at the time of reading configuration wouldn’t be sufficient, since during runtime, a „wild unmatched“ item could appear (because favorites are updated every iteration). Handling this could either mean printing an exception or exiting the application with a non-zero status. Both would not send any mail and isn’t an appropriate solution, in my opinion.